src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private final double side;
+
+    public Square(Color color, int side) {
+        super(color);
+        this.side = side;
+    }
+
+    @Override
+    public double calculateArea() {
+        return side * side;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: Square"
+                + ", area: "
+                + calculateArea() + " sq.units, side: "
+                + side + " units, color: "
+                + getColor().toString().toLowerCase());
```

Comment:
[you need to use method `name()` to get a string representation of the enum. ](https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#use-name-for-getting-string-representation-of-enum-constants)